owned-manifests/baremetal_operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metal3-baremetal-operator
+  namespace: {{ .TargetNamespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: metal3-baremetal-operator
+  template:
+    metadata:
+      labels:
+        name: metal3-baremetal-operator
+    spec:
+      serviceAccountName: metal3-baremetal-operator
+      containers:
+        - name: baremetal-operator
+          image: {{ .BaremetalOperatorConfig.BaremetalOperator }}
+          ports:
+          - containerPort: 60000
+            name: metrics
+          command:
+          - /baremetal-operator
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "baremetal-operator"
+            - name: DEPLOY_KERNEL_URL
+              value: "http://172.22.0.1/images/ironic-python-agent.kernel"
+            - name: DEPLOY_RAMDISK_URL
+              value: "http://172.22.0.1/images/ironic-python-agent.initramfs"
+            - name: IRONIC_ENDPOINT
+              value: "http://localhost:6385/v1/"
+            - name: IRONIC_INSPECTOR_ENDPOINT
+              value: "http://localhost:5050/v1/"
+        # Temporary workaround to talk to an external Ironic process until Ironic is running in this pod.
```

Comment:
Note that this workaround needs to be gone before this PR can go in.  We might want to add an extra note in this file that says that while it's still WIP

Reply:
Yes. Adding that to my TODO list.